src/packages/ratmama/ratmama_parser.py

```diff
@@ -19,8 +19,10 @@
 from ..rescue import Rescue
 from ..rules import rule
 from ..utils import Platforms
+from ..galaxy.galaxy import Galaxy
 
 LOG = logging.getLogger(f"mecha.{__name__}")
+Galaxy = Galaxy()
```

Comment:
[style][error] instance of an object should be lower case, further this shadows name from containing scope (`Galaxy` already defined from  import)

Reply:
As this is a module-level constant, don't you mean upper case?

Reply:
I agree it should be uppercase since it's a module level constant, even if it's a temporary one.

Reply:
yes. the original naming made it confusing to tell if it was constant or not. if its a constant then yes, UPPER_CASE is correct